elasticsearch/Dockerfile.centos7

```diff
@@ -0,0 +1,52 @@
+FROM centos:centos7
```

Comment:
the convention for s2i is that origin is `Dockerfile` and rhel is `Dockerfile.rhel7`. Can we switch this to follow their structure?